pkg/controllers/management/feature/feature_handler.go

```diff
@@ -40,6 +44,17 @@ func (h *handler) sync(_ string, obj *v3.Feature) (*v3.Feature, error) {
 	if obj.Name == features.TokenHashing.Name() {
 		return obj, h.refreshTokens()
 	}
+
+	if obj.Name == features.Harvester.Name() {
+		lister := h.mgmt.Management.NodeDrivers("").Controller().Lister()
+		cli := h.mgmt.Management.NodeDrivers("")
+		m, _ := lister.Get("", obj.Name)
+		val := feature.GetEffectiveValue(obj)
+		m.Spec.Active = val
+		if _, err := cli.Update(m); err != nil {
+			return obj, err
+		}
```

Comment:
Can we wrap these in a standalone function and use `features.GetFeatureByName` instead of `feature.GetEffectiveValue`

Reply:
@gitlawr updated. Thx for the input.